src/main/java/raythx98/duke/command/AddCommand.java

```diff
@@ -0,0 +1,14 @@
+package raythx98.duke.command;
+
+import raythx98.duke.exception.DukeException;
+import raythx98.duke.storage.Storage;
+import raythx98.duke.task.TaskList;
+import raythx98.duke.ui.Ui;
+
+public abstract class AddCommand extends Command {
+    protected final String taskDescriptionDeadline;
+    public AddCommand(String taskDescriptionDeadline) {
+        this.taskDescriptionDeadline = taskDescriptionDeadline;
+    }
+    public abstract void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException;
```

Comment:
Should this public method come along with a Javadoc comment? I noticed similar issues in other classes as well.

Reply:
Agreed, will resolve for all cases, thank you for pointing it out!